src/qt/overviewpage.cpp

```diff
@@ -283,7 +283,7 @@ void OverviewPage::on_btnWebsite_pressed()
 
 void OverviewPage::on_btnBX_pressed()
 {
-    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/"));
+    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/#pk_campaign=GridcoinWallet&pk_kwd=" + QString::fromStdString(FormatFullVersion())));
```

Comment:
I had put it there as when ur in https://gridcoinstats.eu/block and select a block it goes to https://gridcoinstats.eu/block/902102 . put it that way to minimize conflicts if he decides in future for force the / to be there after block in the link. https://gridcoinstats.eu/block and https://gridcoinstats.eu/block/ both load the same.

Ive asked startailcoon to come chime in so he knows how it is and if he wants it different I can accommodate. Let get his part on this to minimize the likely hood of broken link in future.